kubejs/assets/gtceu/lang/zh_cn.json

```
@@ -0,0 +1,49 @@
+{
+    "block.gtceu.neocube": "NeoForging Cuboid",
+    "gtceu.neocube": "NeoCube",
+    "block.gtceu.neural_node": "Neural Network Node",
+    "gtceu.neural_node": "Neural Network Node",
+    "block.gtceu.ore_processing_plant": "Ore Processing Plant",
+    "gtceu.ore_processing_plant": "Ore Processing Plant",
+    "gtceu.ore_bathing_plant": "Ore Chemical Washing Plant",
+    "gtceu.ore_sifting_plant": "Ore Sifting Plant",
+    "block.gtceu.apiary_i": "养蜂场 MK I",
+    "gtceu.apiary_i": "养蜂场 MK I",
+    "block.gtceu.apiary_ii": "养蜂场 MK II",
+    "gtceu.apiary_ii": "养蜂场 MK II",
+    "block.gtceu.comb_processor": "蜂巢处理器",
+    "gtceu.comb_processor": "蜂巢处理器",
+    "block.gtceu.star_forge":"星辰锻炉",
+    "block.gtceu.micro_universe_orb":"微型宇宙球",
+    "block.gtceu.mega_fusion_reactor":"巨型聚变反应堆",
+    "gtceu.star_forge":"星辰锻炉",
+    "gtceu.micro_universe_reactor":"微型宇宙反应堆",
+    "gtceu.micro_universe_collector":"微型宇宙采集器",
+    "gtceu.mega_fusion_reactor":"巨型聚变反应堆",
+    "material.gtceu.inert_nether_essence":"惰化下界之星精华液",
+    "material.gtceu.trinaquadalloy": "钍瑞铌合金",
+    "material.gtceu.fluorite": "氟石",
+    "material.gtceu.dolomite": "白云石",
+    "material.gtceu.star_matter": "星辰物质",
+	"block.gtceu.advanced_large_chemical_reactor": "高级大型化学反应釜",
+	"material.gtceu.nitinol": "镍钛诺",
+    "block.gtceu.greenhouse": "温室",
+    "gtceu.greenhouse": "温室",
+    "block.gtceu.void_miner": "虚空采矿场",
+    "gtceu.void_miner": "虚空采矿场",
+    "block.gtceu.world_data_scanner": "维度扫描器",
+    "gtceu.world_data_scanner": "维度扫描器",
+    "block.gtceu.lv_world_data_scanner": "基础 维度扫描器",
```

```suggestion
    "block.gtceu.lv_world_data_scanner": "基础维度扫描器",
```

Unwanted spaces here and ones down below